client/src/components/ColorSelector.jsx

```diff
@@ -1,17 +1,33 @@
 import React from 'react';
 import ColorButton from './ColorButton.jsx';
-import {ColorContainer, RadioButtonLabel} from '../styling.jsx';
+import {RadioButtonLabel} from '../styling.jsx';
 import styled from 'styled-components';
 
+const ColorContainer = styled.div`
+  background: rgb(250,250,250);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding: 0.5em;
+  background: white;
+  margin: 0px 0px 30px;
+  border-top: 1px solid rgba(0, 0, 0, .3);
+`;
+
 const ColorSelector = (props) => {
   return (
     <ColorContainer className={props.className}>
 
       {props.colors.map((color) => {
-        return <ColorButton key={color} color={color} selectColor={props.selectColor}/>
+        return <ColorButton
+        key={color}
```

Comment:
Not sure if the key needs to be more unique once we put everything together (I know it only important for performance reasons). but perhaps something like `${id}{color}`?

Reply:
good call, I'm going to add this to my list of refactors. I probably won't change it until my proxy is running though